dashboard/client/src/pages/node/index.tsx

```diff
@@ -345,7 +350,15 @@ const Nodes = () => {
                   .map((node) => (
                     <NodeRows
                       key={node.raylet.nodeId}
-                      node={node}
+                      node={
+                        nodeLogicalResourceMap
+                          ? {
+                              ...node,
+                              logicalResource:
```

Comment:
```suggestion
                              logicalResources:
```

Reply:
@alanwguo Would you like to clarify whether "logicalResources" is intended for multiple nodes or for a single node in this context? Or are you trying to convey that one node can have multiple resources (such as CPU, memory, etc.)?

I understand that for the API router, we need to use @routes.get("/nodes/logical_resources") since we fetched logical resources for multiple nodes. However, the logicalResource passed here is intended for a single node.

Reply:
one node can have multiple resources so it should be plural.